Firmware/ChameleonMini/Terminal/Commands.h

```diff
@@ -63,9 +63,9 @@
 
 // Firmware version details.
 #define FIRMWARE_VERSION_MAJOR          1
-#define FIRMWARE_VERSION_MINOR          3
+#define FIRMWARE_VERSION_MINOR          4
 #define FIRMWARE_NAME                   "rebooted"
-#define FIRMWARE_FORK_AUTHOR            "Iceman"
+#define FIRMWARE_FORK_AUTHOR            "Nitro"
```

Comment:
don't.

Reply:
Sorry, forgot to  remove this patch...
I've put this on my fork so I can see which version is running on my chameleon